catalog/spatial/kml/spatial-kml-transformer/src/main/java/org/codice/ddf/spatial/kml/converter/KmlFeatureToJtsGeometryConverter.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) Codice Foundation
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ **/
+package org.codice.ddf.spatial.kml.converter;
+
+public class KmlFeatureToJtsGeometryConverter {
+    private KmlFeatureToJtsGeometryConverter() {
+    }
+
+    public static com.vividsolutions.jts.geom.Geometry from(
+            de.micromata.opengis.kml.v_2_2_0.Feature kmlFeature) {
+        if (kmlFeature == null) {
+            return null;
+        }
+
+        if (kmlFeature instanceof de.micromata.opengis.kml.v_2_2_0.Document) {
+            return KmlDocumentToJtsGeometryConverter.from((de.micromata.opengis.kml.v_2_2_0.Document) kmlFeature);
```

Comment:
Is there a reason to include the whole package for these objects, or can `de.micromata.opengis.kml.v_2_2_0` be dropped?

Reply:
No reason. Dropped.